vbeam/data_importers/pyuff_importer.py

```diff
@@ -100,12 +100,20 @@ def import_pyuff(
     if numpy.abs(modulation_frequency) == 0:
         receiver_signals = np.array(hilbert(receiver_signals), dtype="complex64")
 
-    receivers = ElementGeometry(
-        np.array(channel_data.probe.xyz),
-        np.array(channel_data.probe.theta),
-        np.array(channel_data.probe.phi),
-    )
-    sender = ElementGeometry(np.array([0.0, 0.0, 0.0], dtype="float32"), 0.0, 0.0)
+    if channel_data.probe.__class__ == 'pyuff_ustb.objects.probes.curvilinear_array.CurvilinearArray':
+        ROC_azimuth = channel_data.probe.radius
+    elif channel_data.probe.__class__ == 'pyuff_ustb.objects.probes.curvilinear_matrix_array.CurvilinearMatrixArray':
+        ROC_azimuth = channel_data.probe.radius_x
+    else:
+        ROC_azimuth = 10
+    ROC_elevation = 10
+
+    probe = ProbeGeometry(ROC = (ROC_azimuth, ROC_elevation))
+    probe.rx_aperture_length_s = (np.max(channel_data.probe.x) - np.min(channel_data.probe.x) , np.max(channel_data.probe.y) - np.min(channel_data.probe.y))
+    probe.tx_aperture_length_s = (np.max(channel_data.probe.x) - np.min(channel_data.probe.x) , np.max(channel_data.probe.y) - np.min(channel_data.probe.y))
```

Comment:
We should pass `rx_aperture_length_s` and `tx_aperture_length_s` as arguments when creating the `ProbeGeometry` object.